INSTALL

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
+#
+# $COPYRIGHT$
+# $HEADER$
+#
+
+
+For More Information
+====================
+
+This file is a *very* short overview of building and installing UCX
+and building MPI/SHMEM libraries with it.  Much more information is
+available on the UCX web site and github wiki
+
+    http://openucx.org/
+
+    -and-
+
+    https://github.com/openucx/ucx/wiki
+
+
+Developer Builds
+================
+
+shell$ ./autogen.sh
+shell$ ./contrib/configure-devel --prefix=$PWD/install-debug
+
+You will need very recent versions of GNU Autoconf, Automake, and
+Libtool. 
```

Comment:
we will put autotools versions check in autogen.sh script instead (as separate PR).